src/thread.c

```diff
@@ -129,7 +129,11 @@ static const char* luv_thread_dumped(lua_State* L, int idx, size_t *l) {
   } else {
     const char* buff;
     luaL_checktype(L, idx, LUA_TFUNCTION);
+#if LUA_VERSION_NUM >= 502
+    lua_getglobal(L, "string");
```

Comment:
Good catch! Yes, I have found in Lua5.1 source, it already have lua_[gs]etglobal. So I will use that :)